tests/bson/bson-timestamp-006.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\BSON\Timestamp constructor requires 64-bit integers to be positive unsigned 32-bit integers (as string)
```

Comment:
Unlike the 003 and 004 tests, these are only using string types. It doesn't make sense to refer to 64-bit integers or restrict the test to 64-bit platforms.

Both of these test cases should just be added into `bson-timestamp-005.phpt`, especially because we do want to test that the full range of 32-bit unsigned integers can be expressed on a 32-bit platform.